packages/connect-common/files/coins.json

```diff
@@ -2636,7 +2680,7 @@
         {
             "blockchain_link": {
                 "type": "blockfrost",
-                "url": ["wss://ada1.trezor.io", "wss://ada2.trezor.io"]
+                "url": ["wss://trezor-cardano-mainnet.blockfrost.io"]
```

Comment:
We don't want these changes, seems like these were recently updated in Suite repo but not in FW repo
https://github.com/trezor/trezor-suite/commit/c6d420cde0d56c35b572551524d7382e69e516d8

Reply:
I see, I just talked to @tomasklim and they were modified manually. I will ignore the changes, and just add the changes I need for this. Soon trezor-common will be the source of truth https://github.com/trezor/trezor-firmware/pull/4530